src/Microsoft.Azure.WebJobs.Logging/Internal/TimeBucket.cs

```diff
@@ -27,5 +29,63 @@ public static long ConvertToBucket(DateTime dt)
             }
             return min;
         }
+
+        // Used to compute epoch. 
+        const long ticksBaseline = 633663648000000000L;
+        const long ticksPerMonth = 26785144663987L;
+        
+        public static DateTime CommonEpoch = new DateTime(0, DateTimeKind.Utc); // maps to epoch 0 
+        public static string CommonEpochSuffix = "common";
+
+        public static long GetEpochNumberFromTable(CloudTable table)
```

Comment:
It will return -1 for the common case. 

---
In reply to: [82907220](https://github.com/Azure/azure-webjobs-sdk/pull/862#discussion_r82907220) [](ancestors = 82907220)